packages/blue-sdk-viem/src/signatures/index.ts

```diff
@@ -0,0 +1,11 @@
+export * from "./manager";
+export * from "./permit";
+export * from "./permit2";
+export * from "./utils";
+export * from "./types";
+
+export * as manager from "./manager";
+export * as permit from "./permit";
+export * as permit2 from "./permit2";
+export * as utils from "./utils";
+export * as types from "./types";
```

Comment:
To me it's just like namespaces
But they have too common names (like `types` which may get overridden by main export from root src)

I may even prefer not to export like this at the root src but it's not that of a big deal if it's only done at the highest level

And this made me think about namespaces: they are good to restrict function and constant definitions to a specific domain, so we don't have to specify the domain in the function's name for example

but here we have `getManagerApprovalMessage` it is clear enough that the domain is manager approval and the function is getMessage
Namespaces would be useful as: `ManagerApproval.getMessage`